test/lib/ufe/testObservableScene.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import os
+import sys
+import unittest
+
+import ufe
+
+class TestObserver(ufe.Observer):
+    def __init__(self):
+        self.add = 0
+        self.delete = 0
+        self.pathChange = 0
+        self.subtreeInvalidate = 0
+        self.composite = 0
+        super(TestObserver, self).__init__()
+
+    def __call__(self, notification):
+        if isinstance(notification, ufe.ObjectAdd):
+            self.add += 1
+        if isinstance(notification, ufe.ObjectDelete):
+            self.delete += 1
+        if isinstance(notification, ufe.ObjectPathChange):
+            self.pathChange += 1
+        if isinstance(notification, ufe.SubtreeInvalidate):
+            self.subtreeInvalidate += 1
+        if isinstance(notification, ufe.SceneCompositeNotification):
+            self.composite += 1
+
+    def notifications(self):
+        return [self.add, self.delete, self.pathChange, self.subtreeInvalidate, self.composite]
+
+class UFEObservableSceneTest(unittest.TestCase):
```

Comment:
You need a skipIf tag here so this test only runs if UFE v >= 2021

Reply:
addressed f4f6bd4a848585dfb332e3ee75ac98923af250b5